src/Splat.SimpleInjector/SimpleInjectorDependencyResolver.cs

```diff
@@ -27,12 +28,21 @@ public SimpleInjectorDependencyResolver(Container container)
         }
 
         /// <inheritdoc />
-        public object GetService(Type serviceType, string contract = null) => _container.GetInstance(serviceType);
+        public object GetService(Type serviceType, string contract = null)
+        {
+            return _container.GetInstance(serviceType);
+        }
 
         /// <inheritdoc />
         public IEnumerable<object> GetServices(Type serviceType, string contract = null) =>
             _container.GetAllInstances(serviceType);
 
+        /// <inheritdoc />
+        public bool HasRegistration(Type serviceType)
+        {
+            return _container.GetCurrentRegistrations().Any(x => x.ServiceType == serviceType);
```

Comment:
Could you use `_container.GetRegistration(serviceType)`?

Reply:
get registration made simpleinjector immutable like the issue in #240

Reply:
ah... oh well makes sense.